event_creation/submission/parsers/dtypes.py

```diff
@@ -39,6 +39,7 @@
         ('host_time',-999,'float64'),
         ('id', '', 'U64'),
         ('position', '', 'U64'),
+        ('stimtag', '', 'U64'),  ## LC: EFRCourier has alternating stimulation frequencies (3, 8Hz)
```

Comment:
if this is frequency, it's already described by other fields here. I.e. "pulse_freq" or "burst_freq", don't remember which.

Reply:
Stimtag field removed.